cljfmt/src/cljfmt/core.cljc

```diff
@@ -306,11 +328,68 @@
       (cond-> (:insert-missing-whitespace? opts true)
         insert-missing-whitespace)
       (cond-> (:indentation? opts true)
-        (reindent (:indents opts default-indents)))
+        (reindent (:indents opts default-indents)
+                  (:alias-map opts {})))
       (cond-> (:remove-trailing-whitespace? opts true)
         remove-trailing-whitespace)))
 
+(defn- top-level-containing-form
+  "Find the top-level form which contains `node`.
+
+   A namespace contains several top-level forms, and the root of a
+  zipper for that namespace will have all those forms as children. This
+  function finds the start of the subtree that contains `node`."
+  [node]
+  (->> node
+       (iterate z/up)
+       (take-while (complement root?))
+       last))
+
+#?(:clj
+   (defn- require-node?
+     [node]
```

Comment:
Can you put the argument list on the same line as the function name, as long as there's room to do so?

Also I tent to use `zloc` elsewhere in the source code to denote an arbitrary zipper location, rather than node. Consistent naming conventions should make the code easier to read.